### DISCUSSION TITLE
uncertainty inversion results using the Levenberg-Marquardt algorithm

### DISCUSSION THREAD

**Discussion by ferryatm** [gimli-org/pyGIMLi#746] [General]
How do you calculate the uncertainty of VES inversion results using the Levenberg-Marquardt algorithm? then how do you see or call the Jacobian and Covariance matrices?

**Comment by halbmy** [comment]
We extended the VES example to show covariance matrix and model uncertainties:
https://dev.pygimli.org/_tutorials_auto/3_inversion/plot_2-dc1dblock.html
Upon release of v1.5.2, this will be on the main website
https://www.pygimli.org/_tutorials_auto/3_inversion/plot_2-dc1dblock.html
